src/risk_data.py

```diff
@@ -53,7 +57,7 @@ def risk_data():
 
     df_risk = pd.merge(uploaded_accounts, df_risk, how="inner", on="DNI")
 
-    df_phone_risk = get_phone_risk(df_risk=df_risk)
+    df_phone_risk = get_phones(df=df_risk)
```

Comment:
If we redefine this function to `get_phones(df: pd.DataFrame, stop: int, colum_name: str)`. Shoukld we pass the `stop` and `column_name` in the arguments?
https://github.com/alquimistas-org/subida_cartera/pull/14/files#diff-92256a9f69e6bb74e01a73cf7fbace84472f5eea4f136547962e9992c59ac1b3R6